statsmodels/tools/eval_measures.py

```diff
@@ -67,6 +68,30 @@ def rmse(x1, x2, axis=0):
     x2 = np.asanyarray(x2)
     return np.sqrt(mse(x1, x2, axis=axis))
 
+def rmspe(y, y_hat, axis=0, zeros=np.nan):
+    """
+    root mean squared percentange error
```

Comment:
Capital letter to start please.

Reply:
I would,but the original script doesn't, so I believe [the proper stylistic guideline](https://google.github.io/styleguide/pyguide.html#4-parting-words) is that Local Style should trump Global Style, so as not to cause unnecessary distraction

Reply:
It doesn't.  Those old ones are just strings in need of change that haven't been fixed yet.  Feel free to change those as well.

Reply:
Done now